Structure_Engine/Query/EmbodiedCarbon.cs

```diff
@@ -30,13 +34,19 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Calculates the total amount of embodied carbon of a Bar by taken the mass of the bar (as section area*length*density) times the EmbodiedCarbon value of the material")]
+        [Input("bar", "The Bar to get the total embodied carbon from")]
+        [Output("emobodiedCarbon","The total embodied carbon of the bar", typeof(Mass))]
         public static double EmbodiedCarbon(this Bar bar)
         {
             return bar.Mass() * bar.SectionProperty.Material.EmbodiedCarbon;
         }
 
         /***************************************************/
 
+        [Description("Calculates the total amount of embodied carbon of a Panel by taken the mass of the panel (as area*thickness*density) times the EmbodiedCarbon value of the material")]
```

Comment:
Calculates the total amount of embodied carbon of a Panel by getting the mass of the Panel (as area*thickness*density) multiplied by the EmbodiedCarbon value of the material